src/corefx/System.Globalization.Native/pal_collation.c

```diff
@@ -133,13 +119,31 @@ bool IsHalfFullHigherSymbol(UChar character)
         || (0xff61 <= character && character <= 0xff65);
 }
 
+static bool AddItem(UCharList* list, size_t* currentCapacity, const UChar item)
```

Comment:
A nit - it seems the currentCapacity could be part of the UCharList too. It seems a bit strange that the size is part of it and the capacity is not.

Reply:
Thanks, I have addressed the feedback. Will continue on lldb hacking after the build.